src/librustc_privacy/lib.rs

```diff
@@ -9,6 +9,7 @@
 
 #[macro_use] extern crate rustc;
 #[macro_use] extern crate syntax;
+#[macro_use] extern crate log;
```

Comment:
Nit: could you remove the log dependency, it's almost unused.

Reply:
It's been a big help to me when working on this - the `debug!` line I added makes it easy to see if my code is even running.